### PR TITLE
DLS-4408: [RP] Play 2.8 library upgrade.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: app/uk/gov/hmrc/fhregistrationfrontend/actions/PageAction.scala [hunk 1]
```diff
@@ -67,23 +66,23 @@ class PageAction[T, V] @Inject()(pageId: String, sectionId: Option[String], jour
...
+      logger.error(s"Not found")
       Left(errorHandler.errorResultsPages(Results.NotFound))
     }
 
   def loadPage[A](request: JourneyRequest[A]): Either[Result, Page[T]] =
     request.journeyState.get[T](pageId) match {
       case Some(page) ⇒ Right(page)
       case None ⇒
-        Logger.error(s"Not found")
+        logger.error(s"Not found")
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/fhregistrationfrontend/actions/PageAction.scala:85]: not specific to this PR, but these generic `Not found` messages are no use :) , i would delete them, but its ok if you want to keep them

File: app/uk/gov/hmrc/fhregistrationfrontend/actions/SummaryAction.scala [hunk 2]
```diff
@@ -57,7 +56,7 @@ class SummaryAction(implicit errorHandler: ErrorHandler, val executionContext: E
     if (journeyState.isComplete)
       Right(true)
     else {
-      Logger.error(s"Bad Request")
+      logger.error(s"Bad Request")
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/fhregistrationfrontend/actions/SummaryAction.scala:59]: same as above

File: app/uk/gov/hmrc/fhregistrationfrontend/actions/UserAction.scala [hunk 3]
```diff
@@ -99,7 +98,7 @@ case class UserAction @Inject()(
   def handleFailure(e: Throwable): Result =
     e match {
       case x: NoActiveSession ⇒
-        Logger.warn(s"could not authenticate user due to: No Active Session " + x)
+        logger.warn(s"could not authenticate user due to: No Active Session " + x)
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/fhregistrationfrontend/actions/UserAction.scala:101]: no need to `+` with string interpolation

File: app/uk/gov/hmrc/fhregistrationfrontend/controllers/FormPageController.scala [hunk 4]
```diff
@@ -93,7 +92,7 @@ class FormPageController @Inject()(
       if (request.lastUpdateTimestamp == lastUpdateTimestamp)
         Future successful Ok(views.confirm_delete_section(pageId, sectionId, lastUpdateTimestamp))
       else {
-        Logger.error(s"Not Found. Expired")
+        logger.error(s"Not Found. Expired")
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/fhregistrationfrontend/controllers/FormPageController.scala:95]: same as above

File: project/plugins.sbt [hunk 5]
```diff
@@ -5,16 +5,10 @@ resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.
...
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.23")
-
-addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.8.1")
-
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
-
-addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
-
-addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.13")
+addSbtPlugin("uk.gov.hmrc"         %   "sbt-auto-build"        % "3.4.0")
```
Comment (sureshhmrc) [inline project/plugins.sbt:8]: 3.5.0 can be used , but not a biggie

File: app/uk/gov/hmrc/fhregistrationfrontend/actions/PageAction.scala (hunk 1 above)
Comment (rambabu-posa) [reply app/uk/gov/hmrc/fhregistrationfrontend/actions/PageAction.scala:85]: mmm exactly am not sure why they are added here. we are really no idea. am happy to delete them.

File: app/uk/gov/hmrc/fhregistrationfrontend/actions/UserAction.scala (hunk 3 above)
Comment (rambabu-posa) [reply app/uk/gov/hmrc/fhregistrationfrontend/actions/UserAction.scala:101]: didn't change any code base as part of this library change. Will try to fix it

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**